tests/testdata/default_config.yaml

```diff
@@ -19,6 +19,9 @@ chat:
   # Filepath of a dialog session file.
   # Default: None
   session:
+  # Custom system prompt. If not provided, it will use the default context.
+  # Default: None
+  system_prompt:
```

Comment:
Is there any way for it to not appear in the config at all? Or is this a limitation of the current config system?

Reply:
I think so, seems  a limitation of the current config system. Tried `exclude=True`, still will appear..

Reply:
lol, yeah, but seems evaluate do it too:'
https://github.com/instructlab/instructlab/blob/5922a76266ad65b4cfc1bb7ffeee9de56dd96df6/tests/testdata/default_config.yaml#L112

Reply:
I saw @alimaredia submit a `redesign config`, maybe he has ideas about this.

Reply:
Yeah I suppose it's fine, it just looks very strange.